File: dataset_reduzido/codenet/Python/Code/s724326345.py
def main():
    A, B, C, X, Y = map(int, input().split())
    ans = 10 ** 10

    for k in range(max(X, Y) + 1):
        total = 2 * C * k + A * max(0, X - k) + B * max(0, Y - k)
        if total < ans:
            ans = total

    print(ans)


if __name__ == "__main__":
    main()

File: output/Llama-2-100/codenet/Python/Java/s724326345.java
public class s724326345 {
    public static void main(String[] args) {
        int A, B, C, X, Y;
        A = Integer.parseInt(sc.nextLine());
        B = Integer.parseInt(sc.nextLine());
        C = Integer.parseInt(sc.nextLine());
        X = Integer.parseInt(sc.nextLine());
        Y = Integer.parseInt(sc.nextLine());

        int ans = (int) (10 * Math.pow(10, 10));

        for (int k = Math.max(X, Y) + 1; k > 0; k--) {
            int total = 2 * C * k + A * Math.max(0, X - k) + B * Math.max(0, Y - k);
            if (total < ans) {
                ans = total;
            }
        }

        System.out.println(ans);
    }
}
